fuzz_test.go

```diff
@@ -42,8 +43,10 @@ func TestFuzz_basic(t *testing.T) {
 	}{}
 
 	failed := map[string]int{}
+	f := New().
+		RandSource(rand.NewSource(10))
```

Comment:
I appreciate the desire to make the tests deterministic, but I'm torn-- I want them to pass with all seeds (and I think they don't right now), I'm worried that sticking to a seed that makes them pass might obscure a legitimate problem. Can you separate these changes into a separate PR so we can get your original fix in sooner? :)

Reply:
Sure can - I included it in this PR as the tests were "randomly" failing

Reply:
removed